internal/wrr/random.go

```diff
@@ -51,27 +57,35 @@ var grpcrandInt63n = grpcrand.Int63n
 func (rw *randomWRR) Next() (item interface{}) {
 	rw.mu.RLock()
 	defer rw.mu.RUnlock()
-	if rw.sumOfWeights == 0 {
+	sumOfWeights := rw.items[len(rw.items)-1].AccumulatedWeight
```

Comment:
This will fail if there's no item, right? (And add a test)

Reply:
fixed